perun-core/src/test/java/cz/metacentrum/perun/core/impl/AttributesManagerImplIntegrationTest.java

```diff
@@ -124,6 +126,34 @@ public void getRequiredAttributes_services_resource_group_ForOneService() {
 		assertThat(attributes).hasSize(1);
 	}
 
+	@Test
+	public void getUninitiatedAttributesModule() throws Exception {
```

Comment:
Once method name is fixed to ``uninitialized``, you should also update tests names.

Reply:
Fixed.